gulpfile.js

```diff
@@ -52,19 +52,23 @@ gulp.task('apidoc', (done) => {
   }, done);
 });
 
-gulp.task('test', () => {
-  return gulp.src('./dist-server/test/node/*.js')
+gulp.task('test:index', () => {
+  return gulp.src('./dist-server/test/index.js')
     .pipe(tape({
       reporter: faucet(),
     }));
 });
 
+gulp.task('test', () => {
+  runSequence('babel','test:index');
```

Comment:
index.js intergrate all js files in test/node folder and run all test. If you want to run test seperately, type like `tape test dist-server/test/node/user.js`